test/integration/targets/bigip_device_info/tasks/issue-01364.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Issue 01364 - Select users facts
+  bigip_device_info:
+    include:
+      - users
+  register: result
+
+# - name: Issue 00082 - Assert Select interface facts
```

Comment:
incorrect checks, complete the test, i would suggest, tests like this: create a resource and then get it with facts and verify it is created

Reply:
Fixed up the checks. Added the check as you suggested.